quetz/tasks/workers.py

```diff
@@ -64,10 +63,12 @@ def job_wrapper(
 
     callable_f: Callable = pickle.loads(func) if isinstance(func, bytes) else func
 
-    logger = logging.getLogger("quetz")
-    logger.debug(
-        f"evaluating function {callable_f} in a subprocess task with pid {os.getpid()}"
-    )
+    # import os
```

Comment:
remove?

Reply:
I am still wondering if this is running "too often" per task? I would like to check that